packages/documentation/src/components/pageFooter/Components.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Link, StaticQuery, graphql } from 'gatsby';
+
+/**
+ * Sidebar
+ *
+ * @param {props} props
```

Comment:
Should this be an object?
```suggestion
 * @param {object} props
```
Also, you can specify properties on the object to provide more clarity:
```
@param {string} thing.propertyOfThing
```